docs/README.md

```diff
@@ -5,7 +5,7 @@ Static content delivery is handled using `vercel`.
 
 > Note: The documentation within this repo is specifically focused on the
 > Solana validator client maintained by Solana Labs. The more "common"
-> documentation which is generalize to the Solana protocol as a whole, and apply
+> documentation which is generalized to the Solana protocol as a whole, and apply
 > to all Solana validator implementations, are maintained within the
```

Comment:
```suggestion
> documentation, which is generalized to the Solana protocol as a whole and applies
> to all Solana validator implementations, is maintained within the
```
Thanks! Do you mind merging my suggestion to fix a couple other grammar errors while you're here?